### PR TITLE
Amls 4545

### PR DESCRIPTION
**Pull request by owen-opencast**
Added back links to the MSB section and updated the relevant specs.

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [ ] Appropriate labels added.

### CODE REVIEW

File: test/views/msb/expected_throughputSpec.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@ package views.msb
 
 import forms.{Form2, InvalidForm, ValidForm}
 import jto.validation.{Path, ValidationError}
-import models.moneyservicebusiness.ExpectedThroughput
+import models.moneyservicebusiness.{CETransactionsInNext12Months, ExpectedThroughput}
```
Comment (matthewharris1) [inline test/views/msb/expected_throughputSpec.scala]: Do you need this CETransactions import in this Spec?

File: test/views/msb/funds_transferSpec.scala [hunk 2]
```diff
@@ -18,7 +18,7 @@ package views.msb
 
 import forms.{Form2, InvalidForm, ValidForm}
 import jto.validation.{Path, ValidationError}
-import models.moneyservicebusiness.{FundsTransfer, TransactionsInNext12Months}
+import models.moneyservicebusiness.{ExpectedThroughput, FundsTransfer}
```
Comment (matthewharris1) [inline test/views/msb/funds_transferSpec.scala]: ^ same with Expectedthroughput import

File: test/views/msb/expected_throughputSpec.scala (hunk 1 above)
Comment (owen-opencast) [reply test/views/msb/expected_throughputSpec.scala]: Removed, thanks

File: test/views/msb/funds_transferSpec.scala (hunk 2 above)
Comment (owen-opencast) [reply test/views/msb/funds_transferSpec.scala]: Removed, thanks

**Pull request merged**